docs/conf.py

```diff
@@ -108,6 +108,7 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'cirq_iqm-doc'
 
+html_favicon = 'images/favicon.ico'
```

Comment:
```suggestion
html_favicon = '_static/images/favicon.ico'
```

Reply:
This is where our other projects store static images, for consistency it'd be good to do it here as well.

Reply:
`html_static_path` should be uncommented as well.

Reply:
I checked the docs, https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_favicon, and they seem to hint that the "correct" place for favicons would be `_static/`. Who knows :)